widgets/social-media-buttons/social-media-buttons.php

```diff
@@ -154,6 +165,13 @@ function get_widget_form(){
 							'0.4' => __( 'Very high', 'so-widgets-bundle' ),
 						),
 					),
+					'mobile_width' => array(
+						'type' => 'text',
+						'label' => __( 'Mobile Collapse Width', 'so-widgets-bundle' ),
+						'description' => __( 'The resolution to treat as a mobile resolution.', 'so-widgets-bundle' ),
+						'default' => ( function_exists( 'siteorigin_panels_render' ) ) ? siteorigin_panels_setting( 'mobile-width' ) : 780,
```

Comment:
Same here again. I don't think it's a good idea to have dependencies on other plugins (even optional). It feels like we have a need for a way to set break points globally, not just on a per widget basis, but for the entire bundle. 🤔

Reply:
I'm going to move this setting to a widget setting. I still think we should do a SiteOrigin Page Builder settings check for that setting as it's a minor thing (it's purely for a default) and it's not like it's a third party plugin - it's ours.

Reply:
Creating a dependency means that a future change in PB could break this code. So if you _really_ want to use it, you'll need to do more checking to avoid that. E.g. you're checking for `siteorigin_panels_render`, but using `siteorigin_panels_setting`. Rather check that `siteorigin_panels_setting` exists. Also make sure that `siteorigin_panels_setting('mobile-width')` returns a valid value.

Reply:
I was checking siteorigin_panels_render based on the layout slider but in hindsight, that makes more sense.

Reply:
Regardless, I don't agree with not using the mobile width but to make things easier for you I'm going to exclude this check.